jdaviz/configs/cubeviz/helper.py

```diff
@@ -16,6 +18,21 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.app.hub.subscribe(self, SliceSelectWavelengthMessage,
                                handler=self.select_wavelength)
+        self.app.hub.subscribe(self, AddDataMessage,
+                               handler=self._set_spectrum_x_axis)
+
+    def _set_spectrum_x_axis(self, msg):
+        if msg.viewer_id != "cubeviz-3":
+            return
+        viewer = self.app.get_viewer("spectrum-viewer")
+        ref_data = viewer.state.reference_data
+        if ref_data and ref_data.ndim == 3:
+            for att_name in ["Wave", "Wavelength", "Freq", "Frequency"]:
```

Comment:
Do people make cubes in wave number, etc? How do these map to supported CTYPE and how many of those we need to actually support?

https://github.com/astropy/astropy/blob/43ee5806e9c6f7d58c12c1cb9287b3c61abe489d/astropy/wcs/wcsapi/fitswcs.py#L88

Reply:
Looking at the glue-astronomy translator, it seems like the other likely options are Wavenumber, Velocity and Energy. I'll add those.